lib/hooks/session/index.js

```diff
@@ -190,21 +190,21 @@ module.exports = function(app) {
       // If using Redis, wait for a connection or error
       if (app.config.session.adapter === 'connect-redis' &&
           // Future-proof by ensuring that there is a client object, and that it is an event emitter
-          sails.config.session.store.client &&
-          sails.config.session.store.client instanceof require('events').EventEmitter) {
+          app.config.session.store.client &&
+          app.config.session.store.client instanceof require('events').EventEmitter) {
         return (function() {
           // If the client is already connected (because it was provided in the config)
           // then just continue
-          if (sails.config.session.store.client.connected) {return cb();}
+          if (app.config.session.store.client.connected) {return cb();}
           // Define an error handler for the Redis client which will stop Sails from lifting
           // by returning an error in the hook callback
           var redisErrorHandler = function(err) {
```

Comment:
It removes the error handler upon successful connection; do you think we need to remove the connection handler on error?